spec/features/visitors/phone_confirmation_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
```

Comment:
This entire spec can also be deleted. We moved these

Reply:
done